<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Визуализация</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="front_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Вид спереди</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="v_front_view"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="back_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>401</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Вид сзади</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="v_back_view"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="global_view">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>821</width>
      <height>791</height>
     </rect>
    </property>
    <property name="title">
     <string>Общее представление</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>801</width>
       <height>761</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="v_global_view">
      <property name="spacing">
       <number>6</number>
      </property>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
